modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -53,10 +60,24 @@ The control plane and compute machines require the following IAM role permission
 * `kms:DescribeKey`
 ====
 
-.Required IAM role permissions for compute instance profiles
+.Default IAM role permissions for worker instance profiles
```

Comment:
Why change to `worker` here? 

`compute:` and `controlPlane:` are being used in `install-config.yaml`, I think everyone knows what does `compute` mean.

Reply:
'worker node' is the term used in the documentation and CLI responses, not sure why configuration uses 'compute'...